src/main/resources/workerScripts/fromMultiplePriorityQueues.lua

```diff
@@ -16,7 +18,7 @@ for q in queues.gmatch(queues, NEXT_QUEUE_REGEX) do
     local status, queueType = next(redis.call('TYPE', queueName))
     local payload
     if queueType == 'zset' then
-        local firstMsg = redis.call('ZRANGE', queueName, '0', '0')
+	    local firstMsg = redis.call('ZRANGEBYSCORE', queueName, '-inf', now, 'WITHSCORES', 'LIMIT' , '0' , '1')
```

Comment:
Wow, first Lua review, @talhasson I think score is defined by client when sending the message to the queue